From dd84d0b9e2671c405941056bdb7fc8f4cd8c1baa Mon Sep 17 00:00:00 2001
From: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
Date: Wed, 26 Apr 2023 15:10:04 -0300
Subject: [PATCH] csi-powerstore: add separator after comments

This patch consists of adding separator between header comments and
the resources.
It's necessary because the helm couldn't render the chart and it
had been fixed by adding this separator that indicates when
the resource starts.

Signed-off-by: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
Signed-off-by: Hediberto Cavalcante da Silva <hediberto.cavalcantedasilva@windriver.com>
---
 helm/csi-powerstore/Chart.yaml                          | 2 ++
 helm/csi-powerstore/templates/controller.yaml           | 2 ++
 helm/csi-powerstore/templates/csidriver.yaml            | 2 ++
 helm/csi-powerstore/templates/driver-config-params.yaml | 2 ++
 helm/csi-powerstore/templates/node.yaml                 | 2 ++
 5 files changed, 10 insertions(+)

diff --git a/helm/csi-powerstore/Chart.yaml b/helm/csi-powerstore/Chart.yaml
index 84aede7..73dd269 100644
--- a/helm/csi-powerstore/Chart.yaml
+++ b/helm/csi-powerstore/Chart.yaml
@@ -14,6 +14,8 @@
 #
 #
 
+---
+
 name: csi-powerstore
 apiVersion: v2
 version: 2.6.0
diff --git a/helm/csi-powerstore/templates/controller.yaml b/helm/csi-powerstore/templates/controller.yaml
index 73211b0..622a3c9 100644
--- a/helm/csi-powerstore/templates/controller.yaml
+++ b/helm/csi-powerstore/templates/controller.yaml
@@ -14,6 +14,8 @@
 #
 #
 
+---
+
 apiVersion: v1
 kind: ServiceAccount
 metadata:
diff --git a/helm/csi-powerstore/templates/csidriver.yaml b/helm/csi-powerstore/templates/csidriver.yaml
index 2cca097..4724c4f 100644
--- a/helm/csi-powerstore/templates/csidriver.yaml
+++ b/helm/csi-powerstore/templates/csidriver.yaml
@@ -14,6 +14,8 @@
 #
 #
 
+---
+
 apiVersion: storage.k8s.io/v1
 kind: CSIDriver
 metadata:
diff --git a/helm/csi-powerstore/templates/driver-config-params.yaml b/helm/csi-powerstore/templates/driver-config-params.yaml
index 0c8f07c..06bad1e 100644
--- a/helm/csi-powerstore/templates/driver-config-params.yaml
+++ b/helm/csi-powerstore/templates/driver-config-params.yaml
@@ -14,6 +14,8 @@
 #
 #
 
+---
+
 apiVersion: v1
 kind: ConfigMap
 metadata:
diff --git a/helm/csi-powerstore/templates/node.yaml b/helm/csi-powerstore/templates/node.yaml
index a5f2b29..c06ef03 100644
--- a/helm/csi-powerstore/templates/node.yaml
+++ b/helm/csi-powerstore/templates/node.yaml
@@ -14,6 +14,8 @@
 #
 #
 
+---
+
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-- 
2.25.1
